ansible/templates/minigraph_meta.j2

```diff
@@ -26,6 +26,19 @@
             <a:Reference i:nil="true"/>
             <a:Value>Profile0</a:Value>
           </a:DeviceProperty>
+{% if 't2' in topo %}
+{% if card_type is not defined or card_type != 'supervisor' %}
+          <a:DeviceProperty>
+            <a:Name>MacSecEnabled</a:Name>
+            <a:Reference i:nil="true"/>
+{% if dut_index|int == 0 %}
```

Comment:
is it worth adding `or macsec_card == True` here too?

Reply:
provided index == 0 will always be the uplink card (and theres only one) and we are happy to bake it in, lgtm